modules/hitlnext/src/backend/webhook.ts

```diff
@@ -0,0 +1,52 @@
+import axios, { AxiosError, AxiosRequestConfig } from 'axios'
+import * as sdk from 'botpress/sdk'
+import { backOff } from 'exponential-backoff'
+import { MODULE_NAME } from '../constants'
+
+const MAX_ATTEMPTS = 10
+
+class WebHookService {
+  constructor(private bp: typeof sdk) {}
+
+  public async send(data: { payload: any; botId: string; type: string }) {
+    const config = (await this.bp.config.getModuleConfig(MODULE_NAME)).eventsWebHook
+    if (config) {
```

Comment:
Instead of checking if `config` is defined to know if you should send the data through the webhook, I think you should add a class property, something like `disabled` (e.g. `private disabled = !!this.config`) that you can reuse.